.travis.yml

```diff
@@ -12,7 +12,7 @@ install:
     - "pip install selenium==2.53.0"
     - "pip uninstall -y xblock-drag-and-drop-v2"
     - "python setup.py sdist"
-    - "pip install dist/xblock-drag-and-drop-v2-2.0.15.tar.gz"
+    - "pip install dist/xblock-drag-and-drop-v2-2.0.16.tar.gz"
```

Comment:
Note: this was added to be able to build on to the sandbox and might be removed into another PR when we are ready to release